lib/config/state/screens/s3_fetch.ex

```diff
@@ -0,0 +1,66 @@
+defmodule ScreensConfig.State.Screens.S3Fetch do
+  @moduledoc false
+
+  require Logger
+  alias ScreensConfig.Config
+
+  @behaviour ScreensConfig.State.Fetch
+
+  @impl true
+  def fetch_config(current_version) do
+    with {:ok, body, new_version} <- get_config(current_version),
+         {:ok, parsed} <- Jason.decode(body) do
+      {:ok, Config.from_json(parsed), new_version}
+    else
+      :unchanged -> :unchanged
+      _ -> :error
+    end
+  end
+
+  @impl true
+  def get_config(current_version \\ nil) do
+    bucket = Application.get_env(:screens, :config_s3_bucket)
```

Comment:
A thought on all these calls to `Application.get_env/2` - does having it hardcoded to access the `:screens` config key mean that we need to add new config values to Screenplay with `:screens` as the root key in order to make this code work from there?

Reply:
It does yes. That brings up a good note that we should probably both change the config key to `:screens_config` and document that requirement in the README. I'll work on that now.